js/functionality.js

```diff
@@ -0,0 +1,150 @@
+$(document).ready(function(){
+
+
+	// Detect browser
+	var isFirefox = typeof InstallTrigger !== 'undefined';
+	var isSafari = /constructor/i.test(window.HTMLElement) || (function (p) { return p.toString() === "[object SafariRemoteNotification]"; })(!window['safari'] || (typeof safari !== 'undefined' && safari.pushNotification));
+
+	if(isFirefox == true || isSafari == true){
+		$("[class^='background_section'], #planet, #sun, #city_block").css("transition", "all ease-in-out 0.25s");
+	}
+	else{
+		$("[class^='background_section'], #planet, #sun, #city_block").css("transition", "none");
+	}
+
+	// Detect OS
+	var platform = window.navigator.platform;
+    var  os = null;
+	if (!os && /Linux/.test(platform)) {
+		window.scrollTo(0, 0);
+	}
+
+
+	var currentAnimation = "before";
+	// capture scroll any percentage
+	$(window).scroll(function(){
+		let wintop = $(window).scrollTop();
+		let  scrolltrigger = parseInt($("#services_anchor").offset().top);
```

Comment:
on page loading I reach this with scrolltrigger == 0 (so topbar is shown but should not)